gradle.properties

```diff
@@ -14,6 +14,8 @@ ROPSTEN_NETWORK_BACKEND_BASE_HOST=https://apichain-dev.blockchainds.com
 MAIN_NETWORK_BACKEND_BASE_HOST_WS=https://api.blockchainds.com
 ROPSTEN_NETWORK_BACKEND_BASE_HOST_WS=https://api-dev.blockchainds.com
 
+APTOIDE_PACKAGE_NAME="cm.aptoide.pt"
+APTOIDE_PACKAGE_NAME_DEV="cm.aptoide.pt.dev"
```

Comment:
Should we use this variable for dev build? Or just remove it since we don't use it?